tests/bluetooth/init/testcase.yaml

```
@@ -341,3 +341,7 @@ tests:
       - nrf52840dk_nrf52840
       - nrf52dk_nrf52832
       - rv32m1_vega_ri5cy
+  bluetooth.init.test_config_bt_recv_workq_bt:
+    extra_args:
+      - CONFIG_BT_RECV_WORKQ_BT=y
+    platform_allow: qemu_cortex_m3
```

Why QEMU and not nRF5x SoC?
```suggestion
    platform_allow:
      - nrf52840dk_nrf52840
```